web/src/message_list.ts

```diff
@@ -55,27 +81,14 @@ export class MessageList {
         // query .data.filter directly.
```

Comment:
This TODO is probably in the wrong place; it should be on the `MessageListView` definition? But maybe we should just do the task it suggests as a follow-up.

Reply:
Resolving the TODO will involve deleting the `collapse_messages` variable here, so this seems like a fine place for the TODO.